config/default.json

```diff
@@ -68,6 +68,7 @@
     "maxWebhooksPerUserPerCollective": 50,
     "searchEmailPerHour": 150,
     "searchEmailPerHourPerIp": 50,
+    "userSigninAttemptsPerHourPerIp": 50,
```

Comment:
@znarf : I wasn't exactly sure on this limit here; do you think 50 is reasonable here? or should we reduce it to less. My gut feeling initially was that something like 20 would be better but wasn't sure if it's too aggressive. 🤔